app/src/main/java/com/example/iphonesearchapi/adapter/ItunesAdapter.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.iphonesearchapi.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.iphonesearchapi.databinding.PartialItunesRowBinding
+import com.example.iphonesearchapi.model.ITunesResponse
+import com.example.iphonesearchapi.model.Result
+
+class ItunesAdapter(  var itunesList: MutableList<Result>) :
+    RecyclerView.Adapter<ItunesAdapter.ViewHolder>() {
+
+
+    class ViewHolder(view:PartialItunesRowBinding): RecyclerView.ViewHolder(view.root){
+
+        val textView: TextView = view.textViewtrackName
+
+    }
```

Comment:
We can name this something better. And also it would be a good idea to extract the viewholder to a separate file